deployment/ansible/otp.yml

```diff
@@ -7,5 +7,4 @@
       apt: update_cache=yes
 
   roles:
-    - { role: "cac-tripplanner.otp"}
```

Comment:
@hectcastro Maybe you have an idea of what the convention should be? I've run into this more than once.